### PR TITLE
Remove package with --interactive

### PR DESCRIPTION
**Pull request by wli3**
a small change but a lot of loc files

continue https://github.com/dotnet/cli/pull/10674

part of https://github.com/dotnet/cli/issues/10529

@nkolev92 please review

### CODE REVIEW

**Comment by wli3** [comment]
One of the feed is missing and cannot get the latest runtime packages

**Comment by wli3** [comment]
@dotnet-bot test this please

**Comment by wli3** [comment]
@dotnet-bot test this please

**Comment by wli3** [comment]
@dotnet-bot test [](http://dotnet-ci.cloudapp.net) ** Debian8.2 x64 Debug Build ** — Build finished.

**Comment by wli3** [comment]
@dotnet-bot Debian8.2 x64 Debug Build

**Comment by wli3** [comment]
@dotnet-bot test Debian8.2 x64 Debug Build

File: src/dotnet/commands/dotnet-add/dotnet-add-package/xlf/LocalizableStrings.de.xlf [hunk 1]
```diff
@@ -82,11 +82,6 @@
         <target state="translated">Für das Projekt "{0}" kann keine temporäre Datei generiert werden. Ein Paketverweis kann nicht hinzugefügt werden. Löschen Sie das temporäre Verzeichnis, und versuchen Sie es noch mal.</target>
         <note />
       </trans-unit>
-      <trans-unit id="CmdInteractiveRestoreDescription">
-        <source>Allows the command to stop and wait for user input or action (for example to complete authentication).</source>
-        <target state="translated">Hiermit wird zugelassen, dass der Befehl anhält und auf eine Benutzereingabe oder Aktion wartet (beispielsweise auf den Abschluss der Authentifizierung).</target>
-        <note />
-      </trans-unit>
```
Comment (nguerrera) [inline src/dotnet/commands/dotnet-add/dotnet-add-package/xlf/LocalizableStrings.de.xlf]: This missed the loc deadline. :(

Comment (nguerrera) [reply src/dotnet/commands/dotnet-add/dotnet-add-package/xlf/LocalizableStrings.de.xlf]: Recycling the translation manually.

**Pull request merged**